Ludum Dare 44/Assets/_Scripts/PlayerMain.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMain : MonoBehaviour
+{
+    // Start is called before the first frame update

```

Comment:
C: Delete all functions and add class coemment above line 5 saying

`// So we can leverage the palye'rs gameobject.transform for tracking`